numpy/meson.build

```diff
@@ -53,6 +53,19 @@ endif
 # (see cibuildwheel settings in pyproject.toml), but used by CI jobs already
 blas_symbol_suffix = get_option('blas-symbol-suffix')
 
+use_ilp64 = get_option('use-ilp64')
+if not use_ilp64
+  # For now, keep supporting this environment variable too (same as in setup.py)
+  # `false is the default for the CLI flag, so check if env var was set
+  use_ilp64 = run_command(py,
+    [
+      '-c',
+      'import os; print(1) if os.environ.get("NPY_USE_BLAS_ILP64", "0") != "0" else print(0)'
+    ],
+    check: true
+  ).stdout().strip() == '1'
+endif
+
```

Comment:
Hmm, OK. Might help to mention which environment variable in the comment.

Reply:
The one right below it, `NPY_USE_BLAS_ILP64`.

Reply:
Yeah, I knew that :) But having it in the comment makes it easier to read.

Reply:
I'll tweak this in a follow-up; I'm sure we're not done with dealing with this bit of code.

Reply:
done in gh-24289